gallery/src/pages/components/ha-alert.ts

```diff
@@ -98,7 +98,9 @@ const alerts: {
     description: "Alert with slotted image",
     type: "warning",
     iconSlot: html`<span slot="icon" class="image"
-      ><img src="https://www.home-assistant.io/images/home-assistant-logo.svg"
+      ><img
+        alt=""
```

Comment:
```suggestion
        alt="Home Assistant logo"
```

This component needs fixing in general because the default SVG icons are not labeled and there's no guarantee the text will contain the given alert level.  That's a fix for #14170.  Also the ARIA role is not always set right, but I digress...